### PR TITLE
[#222] Set ADMIN_KW when user applying metadata is a rodsadmin (4-2-stable)

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

**Review by trel — APPROVED** [review]
awesome

**Comment by korydraughn** [comment]
Cherry-picked single commit from #225.

File: packaging/test_plugin_unified_storage_tiering.py [hunk 1]
```diff
@@ -391,6 +391,57 @@ def test_single_quote_data_name__127(self):
...
+            with session.make_session_for_existing_admin() as admin_session:
+                zone_name = IrodsConfig().client_environment['irods_zone_name']
+
+                with session.make_session_for_existing_user('alice', 'apass', lib.get_hostname(), zone_name) as alice_session:
+                    resc_name = 'storage_tiering_ufs_222'
+                    filename = 'test_file_issue_222'
+
+                    try:
+                        lib.create_local_testfile(filename)
+                        alice_session.assert_icommand(f'iput -R rnd0 {filename}')
```
Comment (korydraughn) [inline packaging/test_plugin_unified_storage_tiering.py:407]: Oh forgot, we can't use f-strings in 4.2.

Will adjust all of these lines.

Comment (korydraughn) [reply packaging/test_plugin_unified_storage_tiering.py]: Updated.

**Review by alanking — APPROVED** [review]
I prefer `str.format`, but this is fine :) Squash and # it.

**Comment by korydraughn** [comment]
I prefer that too. It was just quicker to change to what's in the current PR.

I may tweak it since this isn't urgent.

**Comment by trel** [comment]
Yes, let's get this into a release for 4.2.12.1.

**Comment by korydraughn** [comment]
All tests passed for ub18 and cen7.

A few more tweaks to the test were needed to get things working, but everything is where it needs to be now.

@trel I've made the packages available too.

**Comment by korydraughn** [comment]
I'm going to add one additional commit to bump the version number.

For 4-2-stable, it looks like we leave the package revision number as is and only bump the plugin revision number. That is based on what I see for PREP 4.2.11.0/1.

**Comment by korydraughn** [comment]
Have to rebuild packages and make them available again.

**Comment by korydraughn** [comment]
@trel Packages built and available for publishing.

**Review by alanking — APPROVED** [review]
\# it... I think we will need to rebuild packages due to different sha, no?

**Comment by trel** [comment]
looks good, ship it.

**Comment by korydraughn** [comment]
Will look into the rebuild thing.

**Comment by korydraughn** [comment]
#'d

**Comment by korydraughn** [comment]
Will rebuild after this is merged to be on the safe side.

**Comment by korydraughn** [comment]
Packages have been rebuilt and made available for publishing.

**Pull request merged**